packages/library/src/commonMain/kotlin/io/realm/Ignore.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.FIELD)
+@MustBeDocumented
+/**
+ * Annotation marking a field as ignored inside Realm, meaning that it will not be part of the object's schema.
```

Comment:
```suggestion
 * Annotation marking a field as ignored inside Realm, meaning that it will not be part of the models' schema.
```

I think generally in Java we talk about the Realm Model schema instead of `object schema`